### DISCUSSION TITLE
How do I jump to the last edit location?

### DISCUSSION THREAD

**Discussion by frozenspider** [zed-industries/zed#19629] [Feature Requests]
I'm pretty used to hotkey that jumps to the file and position where the last edit was made. This is available in VS Code (`workbench.action.navigateToLastEditLocation`), JetBrains IDEs (Keymap -> Main Menu -> Navigate -> Last Edit Location) and Eclipse (`Go to Last Edit Location` )
How do I bind it with Zed?
Thanks!

**Comment by opsnull** [comment]
@frozenspider  maybe you can try `"ctrl--": "pane::GoBack"` and `"ctrl-shift--": "pane::GoForward"`, I ofen use them to quickly jump history location.

**Comment by frozenspider** [comment]
@opsnull Yeah I use that too but it's quite different and it supplement "go to last edit location" rather than replace it.

**Comment by thataboy** [comment]
It's not quite the same thing but `ctrl-,` and `ctrl-.` go to next/prev git diff and that can be useful. Oddly, I can't find what the commands those keys bind to.

**Comment by frozenspider** [comment]
Looged an issue for that: https://github.com/zed-industries/zed/issues/19731
